### PR TITLE
Add rosinstall_generator 0.1.18

### PR DESCRIPTION
**Pull request by seanyen**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by seanyen** [comment]
@conda-forge/staged-recipes this one is ready to be merged.

cc @wolfv do you want to be a maintainer?

**Comment by wolfv** [comment]
Yes please!

**Comment by seanyen** [comment]
@wolfv done!

File: recipes/rosinstall_generator/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+test:
+  imports:
+    - rosinstall_generator
+  commands:
+    - rosinstall_generator --help
+
+about:
+  home: http://wiki.ros.org/rosinstall_generator
+  license: BSD
```
Comment (chrisburr) [inline recipes/rosinstall_generator/meta.yaml]: Which BSD licence is it? Can you use the SPDX identifier and also add the licence family attribute

File: recipes/rosinstall_generator/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - rosinstall_generator=rosinstall_generator.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - catkin_pkg >=0.1.28
```
Comment (chrisburr) [inline recipes/rosinstall_generator/meta.yaml]: You probably don’t need all of these in host. This should only be the dependencies required to run the setup.py script.

Generally I start with python and pip then add more if it fails.

**Pull request merged**